src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/lecture-unit-management.component.ts

```diff
@@ -52,23 +57,29 @@ export class LectureUnitManagementComponent implements OnInit, OnDestroy {
     };
 
     // Icons
-    faTrash = faTrash;
-    faPencilAlt = faPencilAlt;
+    readonly faTrash = faTrash;
+    readonly faPencilAlt = faPencilAlt;
     faEye = faEye;
```

Comment:
```suggestion
    readonly faEye = faEye;
```
I think you missed this one

Reply:
not my code but we can add it yes